lib/rambling/trie.rb

```diff
@@ -1,11 +1,27 @@
 # frozen_string_literal: true
 
-%w(
-  comparable compressible compressor configuration container enumerable
-  inspectable invalid_operation readers serializers stringifyable nodes
-  version
-).each do |file|
-  require File.join('rambling', 'trie', file)
+if RUBY_ENGINE == 'opal'
+  require 'rambling/trie/configuration'
```

Comment:
Can't we just do:
```ruby
%w(
     comparable compressible compressor configuration container enumerable
     inspectable invalid_operation readers serializers stringifyable nodes
     version
).each do |file|
  require "rambling/trie/#{file}"
end
```
?

Reply:
since Opal doesn't support dynamic require, it's impossible as of now

Reply:
Ah okay.